components/organisms/messaging/components/fullPreview/arrow.js

```diff
@@ -0,0 +1,34 @@
+import styled, { css } from 'styled-components'
```

Comment:
What's your opinion about having our own font library with these files? Or at least move all of them to a separate directory in the `components/icons`?